mirage/scenarios/registrations.ts

```diff
@@ -54,6 +54,14 @@ export function registrationScenario(
 
     server.create('registration', { id: 'beefs' });
 
+    server.create('resource', {
+        name: 'Code',
+        description: 'Code Badge',
+        dateCreated: new Date(),
+        dateModified: new Date(),
+        finalized: true,
+    });
```

Comment:
it looks like this isn't being used either, so we should remove it

Reply:
Ok